dl/src/main/scala/com/intel/analytics/bigdl/dataset/Transformer.scala

```diff
@@ -119,16 +123,37 @@ class SampleToBatch[T: ClassTag]
             sample.copyToLabel(labelData, i*labelLength, labelLength)
             sample.copyToFeature(featureData, i*featureLength, featureLength)
 
+            featureTable(i + 1) = Tensor()
```

Comment:
The table backend is a hashmap. Is it effecient? should we implement a table with an array backend?